modules/ROOT/pages/6.0-release-notes-events.adoc

```diff
@@ -0,0 +1,100 @@
+
+
+[[new-and-improved-events]]
+=== New and improved events
+
+
+[[autocompleter]]
+==== `AutocompleterStart`, `AutocompleterUpdate` and `AutocompleterEnd`
+
+As part of moving `autocompleter` from the default `silver` theme to the {productname} core, three new events are now available: `AutocompleterStart`, `AutocompleterUpdate` and `AutocompleterEnd`.
+
+Themes can act on the `AutocompleterStart` and `AutocompleterUpdate` events and can call `AutocompleterEnd` to cancel the auto completer.
+
+
+[[before-set-content]]
+==== `BeforeSetContent`
+
+The `BeforeSetContent` event now includes the actual serialized content when passing in an `AstNode` to the `editor.setContent`.
+
+
+[[get-content]]
+==== `GetContent`
+
+The `GetContent` event now always passes a `string` for the `content` property.
+
+
+[[theme-load-error]]
+==== `ThemeLoadError`
+
+A `ThemeLoadError` event is now fired if the theme fails to load.
+
+
+[[fixed-events]]
+=== Fixed events
+
+
+[[before-input]]
+==== `beforeinput`
+
+The `beforeinput` event is expected to fire whenever material is added to or removed from a {productname} instance.
+
+Previously, however, because of overrides in the `DeleteBackspaceKeys.ts` module, the `beforeinput` event did not always fire when expected.
+
+With this release, these over-rides no longer prevent `beforeinput` events firing when expected.
+
+
+[[change]]
+==== `change`
+
+Previously, the `change` event was triggered by the first modification made to a {productname} editor instance.
+
+However, subsequent changes did not cause the event to fire until focus was switched away from the editor.
+
+For example, typing a single character in to the {productname} editor text-entry area triggered the `change` event.
+
+Consequent typing, or selecting of the entered character and modifying it by setting it to *bold* or _italic_, did not trigger further `change` events.
+
+Making further modifications and then switching focus away from the editor did trigger this event, however.
+
+As of {productname} 6, the `change` event is not fired unless and until
+
+. focus is switched away from the editor; and
+. changes have been made in the editor since focus was switched to it.
+
+NOTE: it is still possible to listen for modification to a {productname} editor instance before focus is changed. To do this, listen for the `dirty` event, rather than the `change` event.
+
+[[fixed-get-content]]
+==== `GetContent`
+
+Previously the `GetContent` event did not fire when the `editor.selection.setContent()` API attempted to get `tree` or `text` formatted data.
+
+With this release, the `GetContent` event fires as expected when `editor.selection.setContent()` attempts this.
+
+NOTE: The `editor.selection.setContent()` API only supports the `html` and `raw` formats. It does not support `tree` or `text` formats. To create a tree to pass in to `editor.selection.setContent()`, use the `new tinymce.html.Node()` API, or parse to a tree using `editor.parser.parse(html)`.
```

Comment:
This doesn't make sense? We say it doesn't support the `tree` format, but then we tell them to create one and use it with the API? I think we should just drop the last sentence entirely.